dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -821,38 +821,52 @@ def publish_docs(
     "-a",
     "--airflow-site-directory",
     envvar="AIRFLOW_SITE_DIRECTORY",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, resolve_path=True),
     help="Local directory path of cloned airflow-site repo.",
     required=True,
 )
 @click.option(
     "-g",
     "--gen-type",
-    help="Type of back references to generate, supports: [airflow | providers | helm]",
-    type=str,
-    required=True,
+    show_default=True,
+    help="Type of back references to generate. Forced to providers if providers specified as arguments.",
```

Comment:
This helper text is not so clear. 
Can we instead do:
"Type of back references to generate. Defaults to all providers if providers are provided as argument"

Reply:
Well. It is actually foreced, not defaulted. Even if you specify providers as parameter (and you get warning about it). . So I think "forcing" is better.